### ISSUE TITLE
Post and reply using threads

### ISSUE THREAD

**Issue by Mattis142** [LucasGGamerM/moshidon#143]
Twitter allows you to Post threads (and also has a hidden feature flag to make reply threads) I think it'd be useful for instances with the default 500 character limit to make a very similar feature. maybe even with an option to automatically add a next reply if the character limit of the first one got exceeded without the need to press a (+) icon
![Screenshot_20230331-132536_SystemUI.png](https://user-images.githubusercontent.com/40364718/229108710-a4e8b0dc-ad16-4b3e-b8e0-8321fbbf0159.png)

**Comment by Andrewblasco** [comment]
Yeah, maybe if you exceeds the limit, Moshidon could suggest you to divide in two posts, or something like that? Umm I'd would be awesome some changes to ease write threads.
